epfast-annotations/src/main/java/com/exactpro/epfast/CreatorService.java

```diff
@@ -7,15 +7,19 @@
 import static com.exactpro.epfast.util.Stream.streamIterable;
 
 public class CreatorService {
-    private static Collection<ICreator> providers = getProviders();
-
-    private static Collection<ICreator> getProviders() {
-        return streamIterable(ServiceLoader.load(ICreator.class)).collect(Collectors.toList());
-    }
 
     public static ICreator getCreator(String fastPackage) {
-        return providers.stream().filter(iCreator -> iCreator.getFastPackage().equals(fastPackage))
+        return InitializationOnDemand.providers.stream()
+            .filter(iCreator -> iCreator.getFastPackage().equals(fastPackage))
             .findFirst().orElseThrow(() ->
                 new RuntimeException(String.format("Creator with FastPackage=\"%s\" not found", fastPackage)));
     }
+
+    private static class InitializationOnDemand {
```

Comment:
Move to separate PR. It's isn't lazy initialization. It will load providers on class load, not at time `getCreator()` is called